admin-tools/data_mover.sh

```diff
@@ -0,0 +1,116 @@
+#! /bin/bash
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/2d3b8cb16fc19c333293924dc7131630e0c50f83